UBI7-init/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi7-init
+MAINTAINER https://github.com/JacobCallahan
+LABEL broker_compatible="True"
+
+ENV INIT_ENABLED 1
+ENV HOME /root
+WORKDIR /root
+
+# set root password
+ARG ROOT_PASSWD=change-me
+RUN echo "root:${ROOT_PASSWD:-default}" | chpasswd
```

Comment:
Wouldn't this be a security threat to echo the password in such a way? Especially if the dockerfile started outside of SatQE team ?

Reply:
Can you please elaborate on how would a malicious entity get the password?

running `podman image inspect` on the image built with this Dockerfile, I can not see the password being exposed.

```json
{
        "created": "2024-06-27T16:10:00.679006594Z",
        "created_by": "/bin/sh -c #(nop) ARG ROOT_PASSWD",
        "author": "https://github.com/JacobCallahan",
        "comment": "FROM 8611db347597",
        "empty_layer": true
      },
      {
        "created": "2024-06-27T16:10:02.001162264Z",
        "created_by": "|1 ROOT_PASSWD=dog8code /bin/sh -c echo \"root:${ROOT_PASSWD:-default}\" | chpasswd",
        "author": "https://github.com/JacobCallahan",
        "comment": "FROM ff5377ba2422"
      },
```

What changes when the container is run `outside of SatQE team`?

Reply:
@ogajduse Consider the scenario where the password is a secret set by some `authority` but the image build is being done from another entity in the team. Wouldn't echoing that password likely to leak it ? Also in general, echoing the password is a bad practice at all times. 


I am not the security expert but I raised this as a security awareness person so I would leave the decision to you and @JacobCallahan .